### DISCUSSION TITLE
Resize Window

### DISCUSSION THREAD

**Discussion by petrasvestartas** [ocornut/imgui#5563] [New Users Build/Link/Run issues ONLY!]
Hi, 

Is it possible to resize the window of imgui e.g. scale everything 2 times?
It is very small on my screen. 
If yes which property is responsible for that?

![image](https://user-images.githubusercontent.com/18013985/183973103-82887bb9-11b7-4c82-8f52-17336de0f3a0.png)

**Comment by rokups** [comment]
Dear ImGui has a limited support for HDPI screens. You can make life more bearable by scaling up style and font size.

```cpp
float scale_factor = 2.0f;
ImFontConfig config;
config.SizePixels = roundf(13.0f * scale_factor);
io.Fonts->AddFontDefault(&config);
io.Fonts->AddFontFromFileTTF("../../misc/fonts/Roboto-Medium.ttf", roundf(16.0f * scale_factor));
style.ScaleAllSizes(scale_factor);
```

⚠️ Font sizes must be rounded to nearest integer. Fractional font sizes are not supported and cause weird issues.

It won't be perfect, but it goes most of the way.
Lots of related issues and discussions about the topic on issue tracker as well: https://github.com/ocornut/imgui/labels/dpi

**Reply by petrasvestartas** [reply]
Aciu @rokups 

This does the job:
```cpp
	float scale_factor = 2.0f;
	ImFontConfig config;
	config.SizePixels = roundf(13.0f * scale_factor);
	io.Fonts->AddFontDefault(&config);
```

Where does this variable style coming from `style.ScaleAllSizes(scale_factor);` ?
(I am using basic example of glfw)

**Reply by rokups** [reply]
> ImGuiStyle& style = ImGui::GetStyle();

**Reply by petrasvestartas** [reply]
Thank you
